hw/top_earlgrey/data/ip/chip_otp_ctrl_testplan.hjson

```diff
@@ -0,0 +1,227 @@
+
+            Xref'ed with corresponding IP tests that receive these bits.
+            '''
+      features: ["OTP_CTRL.ENTROPY_READ"]
+      stage: V2
+      si_stage: SV1
+      tests: ["chip_sw_lc_ctrl_otp_hw_cfg"]
+    }
+    {
+      name: chip_sw_otp_ctrl_lc_signals
+      desc: '''Verify the broadcast signals from LC ctrl.
+
+            - `lc_creator_seed_sw_rw_en_i`: verify that the SECRET2 partition is locked.
+            - `lc_seed_hw_rd_en_i`: verify that the keymgr outputs a default value when enabled.
+            - `lc_dft_en_i`: verify that the test interface within OTP ctrl is accessible.
+            - `lc_check_byp_en_i`: verify that the background check during LC ctrl state
+              programming passes when enabled.
+
+            Note that `lc_escalate_en_i` is verified via a connectivity test.
+
+            The `lc_seed_hw_rd_en_i` signal can be tested by attempting a keymgr advance operation
+            into the CreatorRootKey state, which should fail since the root key will be tied off to
+            all-zero when the SECRET2 partition is not locked in OTP.
+
+            X-ref'ed with chip_sw_lc_ctrl_broadcast test, which verifies the connectivity of the LC
+            decoded outputs to other IPs.
+            '''
+      features: ["OTP_CTRL.PARTITION.SECRET2"]
+      stage: V2
+      si_stage: SV1
+      tests: [
+        // lc_dft_en_i
+        "chip_prim_tl_access",
+        // lc_check_byp_en_i
+        "chip_sw_lc_ctrl_transition",
+        // lc_seed_hw_rd_en_i, lc_creator_seed_sw_rw_en_i, also checks lc_keymgr_en_i since it uses
+        // the keymgr.
+        "chip_sw_otp_ctrl_lc_signals_test_unlocked0",
+        "chip_sw_otp_ctrl_lc_signals_dev",
+        "chip_sw_otp_ctrl_lc_signals_prod",
+        "chip_sw_otp_ctrl_lc_signals_rma",
+        ]
+    }
+    {
+      name: chip_sw_otp_prim_tl_access
+      desc: '''Verify that the SW can read / write the prim tlul interface.
+
+            - The prim tlul interface is a open source placeholder for the closed source CSRs that
+              will be implemented in a translation 'shim'.
+            - Verify that when `lc_dft_en_i` is On, this region can be read / written to by the SW.
+              When `lc_dft_en_i` is Off, accessing this region will result in a TLUL error.
+            '''
+      stage: V2
+      si_stage: SV3
+      tests: ["chip_prim_tl_access"]
+    }
+    {
+      name: chip_sw_otp_ctrl_vendor_test_csr_access
+      desc: '''
+            Verify the vendor test control access in raw, test_*, dev, prod, and rma LC states.
+
+            - Boot the chip successively in raw, test_*, dev, prod and rma LC states.
+            - Verify that the SW is able to access the vendor test control and status registers in
+              raw, test_* and rma LC states.
+              In open source environment, this check is implemented by probing the OTP_CTRL's
+              `lc_otp_vendor_test_i` port.
```

Comment:
This is not feasible in sival. Can you describe an alternative way to test this? Maybe using DAI?

Reply:
https://github.com/lowRISC/opentitan/pull/19889#discussion_r1374991102